### DISCUSSION TITLE
Update on Adding a Community-wide Discussions?

### DISCUSSION THREAD

**Discussion by ChristopherA** [community/community#28943] [Discussions]
For a brief while, we could designate one repo for our "community" discussions (ours are currently at https://github.com/blockchaincommons/community/discussions), that showed up in our Community as a top-level items, but this feature disappeared recently.

I would like it to point to our community/discussions. 

Any update on when this will be available again?

-- Christopher Allen

P.S. I tried to search for similar discussions but "community discussions" is the name of this community, so finds everything ;-)

**Comment by ChristopherA** [comment]
It looks like this feature is enabled here for this community. How do I get this added to my community?

-- Christopher Allen

**Comment by ettaboyle** [comment]
Hi @ChristopherA 👋🏻 Thanks for sharing this. I think what you're looking for is [Enabling or disabling GitHub Discussions for an organization](https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization) <-- That doc outlines all the steps needed to choose a source repo for the org.

Please let me know if that is what you needed or if you're still stuck.
